### PR TITLE
Update Payment Receipts

### PR DESCRIPTION
**Pull request by bcdevlucas**
*Issue #, if available:*
/bcgov/entity#5318

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: api/namex/models/request.py [hunk 1]
```diff
@@ -279,8 +278,9 @@ def get_inprogress(cls, userObj):
         return existing_nr
 
     @classmethod
-    def find_name(cls, nr, choice):
-        return cls.query.filter_by(nrNum=nr).names.filter_by(choice=choice).one_or_none()
+    def find_name_by_choice(cls, nr_id, choice):
```
Comment (severinbeauvais) [inline api/namex/models/request.py:281]: I like the new name 👍

File: api/namex/resources/payment/payment.py [hunk 2]
```diff
@@ -124,6 +125,25 @@ def handle_auth_error(ex):
     return response
 
 
+def custom_strftime_suffix(d):
+    return 'th' if 11 <= d <= 13 else {1: 'st', 2: 'nd', 3: 'rd'}.get(d % 10, 'th')
```
Comment (severinbeauvais) [inline api/namex/resources/payment/payment.py:129]: If I understand correctly, this returns "1st", "2nd", "3rd" or "Xth".

What does this do for 11, 12, 13?

**Review by severinbeauvais — APPROVED** [review]

**Comment by bcdevlucas** [comment]
@severinbeauvais Can you please merge this I don't have write access.

**Comment by severinbeauvais** [comment]
> @severinbeauvais Can you please merge this I don't have write access to namex, just namerequest for some reason or another.

I don't have write access either.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**